python/ray/experimental/channel/common.py

```diff
@@ -19,6 +19,13 @@
     import torch
 
 
+# Holds the input arguments for an accelerated DAG node.
```

Comment:
Change 4: Move `RayDAGArgs` from `compiled_dag_node.py` to `common.py` to avoid importing it inside `_adapt`.